t/03-proxy_wasm/hfuncs/131-proxy_dispatch_http_timeouts.t

```diff
@@ -31,6 +31,45 @@ qq{
     location /t {
         wasm_socket_connect_timeout 1ms;
 
+        proxy_wasm hostcalls 'test=/t/dispatch_http_call \
+                              host=google.com \
+                              on_http_call_response=echo_response_headers';
+        echo ok;
+    }
+}
+--- response_body_like
+:dispatch_status: timeout
```

Comment:
@hishamhm This version will support `get_http_headers` and return the `:dispatch_status` pseudo-header in the table